CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Recurly PHP Client Library CHANGELOG
 
+## Version 2.12.10 (December 31, 2019)
+
+* Add vat_number to ShippingAddress class [PR] (https://github.com/recurly/recurly-client-php/pull/443)
+* Update the documentation and testing matrix for current PHP releases [PR] (https://github.com/recurly/recurly-client-php/pull/445)
+* Simplify the PHPUnit configuration [PR] (https://github.com/recurly/recurly-client-php/pull/446)
```

Comment:
Remember, the space in b/w the `[]` and `()` is not valid in markdown. Seems like some of the other ones have this problem too.

Reply:
Fixed!